app/src/main/java/com/bucketlist/dao/BucketListItemDatabase.kt

```diff
@@ -4,7 +4,7 @@ import androidx.room.Database
 import androidx.room.RoomDatabase
 import com.bucketlist.dto.BucketListItem
 
-@Database(entities = arrayOf(BucketListItem::class), version = 1)
+@Database(entities = [BucketListItem::class], version = 1)
```

Comment:
This was recommended by Android Studio.
One of the reasons we are using Kotlin is because it saves us extra words, and lines of code. Why write extra when it can be
done in less words.